src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -360,3 +365,15 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 # VPN connection shared key
 register_cli_argument('network vpn-connection shared-key', 'connection_shared_key_name', CliArgumentType(options_list=('--name', '-n')), id_part='name')
 register_cli_argument('network vpn-connection shared-key', 'virtual_network_gateway_connection_name', CliArgumentType(options_list=('--connection-name',), metavar='NAME'), id_part='name')
+
+# Traffic manager profiles
+register_cli_argument('network traffic-manager profile', 'traffic_manager_profile_name', name_arg_type, id_part='name')
+register_cli_argument('network traffic-manager profile', 'profile_name', name_arg_type, id_part='name')
+register_cli_argument('network traffic-manager profile create', 'routing_method', choices=get_enum_choices(routingMethod), completer=get_enum_type_completion_list(routingMethod))
+register_cli_argument('network traffic-manager profile check-dns', 'name', name_arg_type, help='DNS prefix to verify availability for.', required=True)
+register_cli_argument('network traffic-manager profile check-dns', 'type', help=argparse.SUPPRESS, default='Microsoft.Network/trafficManagerProfiles')
```

Comment:
>etwork traffic-manager profile check-dns', 'type', h [](start = 24, length = 52)

Is there another value that could secretly be specified that would be valid, or does this exist simply for the template?